### DISCUSSION TITLE
How to create a Jail running Rocky Linux on TrueNAS Scale 24.04.2?

### DISCUSSION THREAD

**Discussion by ronie-camilo** [Jip-Hop/jailmaker#256] [Q&A]
I would like to know the best way to configure a jail to run Rocky Linux 9 on TrueNAS 24.04.2. Could someone with experience help me with the following questions:
Network configuration: What is the best approach to configure the jail's network? Bridge, VLAN, or other option?
Storage: What type of dataset is recommended for the jail's storage?
Resources: How to efficiently set the CPU and memory resources for the jail?
Package installation: Are there any particularities in installing packages within the jail?
Backups: What is the best way to perform backups of the jail?
Thank you in advance for any tips or tutorials that could help me with this configuration.

**Comment by Lockszmith-GH** [comment]
When the jlmkr.py script processes a template (or command line switches), it uses [run_lxc_download_script](https://github.com/Jip-Hop/jailmaker/blob/18d0991cb3651cd387b9561c1e073903cf5007b5/jlmkr.py#L843) function to do that.
The function pulls a copy of the lxc-download bash script from [here](https://raw.githubusercontent.com/Jip-Hop/lxc/97f93be72ebf380f3966259410b70b1c966b0ff0/templates/lxc-download.in).

Running the following in the cli you'll get a list of available linux distributions available for download:
```shell
sh <(curl -Ls https://raw.githubusercontent.com/Jip-Hop/lxc/97f93be72ebf380f3966259410b70b1c966b0ff0/templates/lxc-download.in) --list
```

You can also see the list in a browser at https://images.lxd.canonical.com.
Rocky Linux is listed.

All you need now is to use that as the distro instead of debian.
